.github/workflows/docker_build_tpls.yml

```diff
@@ -6,11 +6,13 @@ on:
     # No input required
 
 jobs:
-  clang-10:
+  build_images:
     runs-on: ubuntu-latest
-    name: Builds a Clang 10 docker image and extracts generated hostconfigs
+    name: Builds a docker image and extracts generated hostconfigs
```

Comment:
This uses a matrix strategy to reduce boiler plate between docker builds in the github action